src/js/modules/articles/components/summaries/RightTitleArticle.js

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import injectSheet from "react-jss";
+import Row from "react-bootstrap/lib/Row";
+import Col from "react-bootstrap/lib/Col";
+
+import Byline from "../Byline";
+import Dateline from "../Dateline";
+
+const styles = {
+  RightTitleArticle: {
+    borderTop: "1px solid #ddd",
+    borderBottom: "1px solid #ddd",
+    padding: "26px 0 26px 0",
+  },
+  featuredMedia: {
+    "& figure": {
+      margin: 0,
+      maxHeight: "500px",
+      overflow: "hidden",
+      width: "100%",
+      "& img": {
+        width: "100%",
+      },
+    },
+  },
+  article: {
+    marginLeft: "55px",
+    paddingTop: "35px",
+    textAlign: "center",
+    width: "325px",
+    "& a": {
+      color: "#000",
+      "&:hover, &:active, &:focus": {
+        color: "#000",
+      },
+    },
+  },
+  sectionLink: {
+    display: "block",
+    fontFamily: "Circular Std",
+    fontSize: "13px",
+    fontWeight: 300,
+    marginBottom: "8px",
+    textTransform: "uppercase",
+  },
+  title: {
+    display: "block",
+    fontFamily: "Canela",
+    fontSize: "40px",
+    fontWeight: 300,
+    lineHeight: "48px",
+    marginBottom: "13px",
+  },
+  summary: {
+    fontFamily: "Minion Pro",
+    fontSize: "16px",
+    lineHeight: 1.25,
+    marginBottom: "18px",
+  },
+  "@media (max-width: 991px)": {
+    article: {
+      marginLeft: 0,
+      paddingTop: "3vw",
+      width: "41.666666%", // col-sm-5
+    },
+    summary: {
+      margin: "0 auto 18px auto",
+      width: "80%",
+    },
+  },
+  "@media (max-width: 767px)": {
+    featuredMedia: {
+      paddingRight: "0 !important",
+      "& figure": {
+        maxHeight: "50vh",
+      },
+      "& figure img": {
+        marginLeft: "-14px",
+        width: "100vw",
+      },
+    },
+    article: {
+      padding: "14px 0 0 0 !important",
+      width: "100%",
+    },
+    title: {
+      fontSize: "30px",
+      lineHeight: "36px",
+    },
+  },
+};
+
+const RightTitleArticle = ({ classes, article }) => {
```

Comment:
Little confused about this name. It's quite possible there isn't a better option, in which case I'd like a comment explaining what this is. But please do think about alternative names. I can help if you'd like

Reply:
comment would b helpful